src/SprykerEco/Zed/Heidelpay/Business/HeidelpayFacadeInterface.php

```diff
@@ -36,6 +37,54 @@ interface HeidelpayFacadeInterface
      */
     public function postSaveHook(QuoteTransfer $quoteTransfer, CheckoutResponseTransfer $checkoutResponseTransfer): CheckoutResponseTransfer;
 
+    /**
+     * Specification:
+     *  - TBD
```

Comment:
IMO we can't release to clients this way.

Reply:
Added specifications